docs/plugins.md

```diff
@@ -135,6 +143,18 @@ function embed(
 
 If you don't want to switch to a different parser, simply return `null` or `undefined`.
 
+### Utility functions
+
+A `util` module from prettier core is considered a private API and is not meant to be consumed by plugins. Instead, the `util-shared` module provides the following limited set of utility functions for plugins:
```

Comment:
We should note that it is an argument to ` genericPrint`

Reply:
Actually, it isn't anymore - i removed it a later version of the PR because we're passing `options` to every functions it exports - which means one can just import the module directly.

Reply:
Ah cool, that's probably better. We might want to expose it as `require('prettier').util` as we bundle the code before publishing to npm.

Reply:
Right - i think that's why CI is currently failing for my PR in the PHP plugin 😅 Shall i add it to this PR?

Reply:
Sure!